cloudbuild.yaml

```diff
@@ -1,11 +1,4 @@
 steps:
   - name: maven
     entrypoint: mvn
-    args: ['-f', './pamelalozano', 'package']
-  - name: maven
-    entrypoint: mvn
-    args: ['-f', './aroquev/portfolio', 'package']
-  - name: maven
-    entrypoint: mvn
-    args: ['-f', './ayup/portfolio', 'package']
-
+    args: ['-f', './', 'package']
```

Comment:
Can you just add your starters/ directory to the top level here? looks like they are not running in the check

Reply:
Done.